docs/content.zh/docs/deployment/config.md

```diff
@@ -398,7 +398,7 @@ With the introduction of `state.backend.rocksdb.memory.managed` and `state.backe
 
 **Blob Server**
 
-The Blob Server is a component in the JobManager. It is used for distribution of objects that are too large to be attached to a RPC message and that benefit from caching (like Jar files or large serialized code objects).
+The Blob Server is a component in the JobManager and the TaskManager. It is used for distribution of objects that are too large to be attached to an RPC message and that benefit from caching (like Jar files or large serialized code objects).
```

Comment:
Thank you for contributing to Flink. As far as I know, the Blob Server is only located in the JobManager. The Blobs in the TaskManager are stored in BlobCacheService. 

However, I think it's a bit of confusing that the configuration of BlobServer, BlobCacheService and BlobClient are all located at `BlobServerOptions.java` in the code and `Blob Server` section in the documentation. Should we just rename `BlobServerOptions.java` to `BlobOptions.java` and rename `Blob Server` section into `Blob`?